en/exercises/device_behaviour_check/device_assessment_checklist.md

```diff
@@ -0,0 +1,117 @@
+
+#### Device Assessment Checklist
+
+##### OSX
+ * OS Security Updates
+
+ * Firewall
+    * See http://support.apple.com/en-us/HT1810 for cross-version guidance
+    * (GUI) Choose System Preferences from the Apple menu, Security (10.5 and before) or Security & Privacy (10.6 and later), then the Firewall tab.
+ * Anti-Virus Version
+    * 
+ * User privilege
+    * 
+ * Drive Encryption
+   * (CLI) 
+```sudo fdesetup status```
+   * (GUI) Choose System Preferences from the Apple menu, Security (10.5 and before) or Security & Privacy (10.6 and later), then the FileVault tab.
+   * (TrueCrypt)
+ * Services Running
+   * (Command line) 
+```sudo launchctl list```
+   * (Command line) 
+```ps -ef```
+   * (GUI) The "Activity Monitor" application is located in /Applications/Utilities provides a similar interface to "top"
+
+##### Windows
+If Windows is not your primary OS, you can download sample Virtual Machines (with time limitations) from Microsoft through their project to improve IE support via https://www.modern.ie/en-us/virtualization-tools#downloads (see also http://www.makeuseof.com/tag/download-windows-xp-for-free-and-legally-straight-from-microsoft-si/ and https://modernievirt.blob.core.windows.net/vhd/virtualmachine_instructions_2014-01-21.pdf) 
+
+**Windows 8**
```

Comment:
I will recommend adding Windows 10 and removing Windows XP, or recommend removing XP in case it was found on one of the machines.

Reply:
Agree with recommendation and added Windows 10 section, though not sure if it fall matches standard SAFETAG frame. Advise on that. 
Pull request #329 shows change